backend/app.py

```diff
@@ -49,7 +49,7 @@ def acceptUpload():
     try:
         if flask.request.method == 'POST':
             metadata = json.loads(flask.request.form['metadata'])
-            files = sorted(flask.request.files.getlist("files"),
+            files = sorted([file for file in flask.request.files.getlist("files") if file.filename.lower().endswith(('.jpg', '.jpeg', '.tif'))],
```

Comment:
doing this removes the other files that are present. "other_files" folder remains empty. There can be some obs files that can be useful for later

Reply:
So what kind of condition do we need? Just ignore any files starting with `.`?